packages/artillery-plugin-publish-metrics/lib/dynatrace/index.js

```diff
@@ -47,21 +76,45 @@ class DynatraceReporter {
       );
 
       const request = this.formRequest(
-        this.formPayload(counters, rates, summaries)
+        this.formMetricsPayload(counters, rates, summaries)
       );
       await this.sendRequest(this.ingestMetricsEndpoint, request);
     });
+
+    this.startedEventSent = false;
+    if (this.eventConfig && String(this.eventConfig.send) !== 'false') {
+      events.on('phaseStarted', async () => {
+        debug('phaseStarted event fired');
+        if (this.startedEventSent) {
+          return;
+        }
+        const timestamp = Date.now();
+        this.eventOpts.startTime = timestamp;
+        this.eventOpts.endTime = timestamp + 1;
+        this.eventOpts.properties.Phase = 'Test-Started';
+
+        await this.sendRequest(
+          this.ingestEventsEndpoint,
+          this.formRequest(JSON.stringify(this.eventOpts), 'event'),
+          'event'
+        );
+
+        this.startedEventSent = true;
+      });
+    }
   }
 
-  parseDimensions(dimensionList) {
+  parseDimensions(dimensionList, returnValue = 'list') {
```

Comment:
Ah I see. I was trying to not repeat myself as the functions are almost identical but I see your point. I would however call it the `parseProperties` instead of `parseDimensionsAsObject` as it will then be used only on event `properties`.